pac4j-oauth/src/main/java/org/pac4j/oauth/redirect/OAuth20RedirectActionBuilder.java

```diff
@@ -53,7 +53,7 @@ public RedirectAction redirect(final WebContext context) throws HttpAction {
 
                 service = this.configuration.getService();
             }
-            final String authorizationUrl = service.getAuthorizationUrl(this.configuration.getCustomParams());
+            final String authorizationUrl = service.getAuthorizationUrl();
```

Comment:
One last question: why custom parameters are not passed anymore to the `getAuthorizationUrl()` method?